core/src/main/java/org/frankframework/filesystem/SftpFileSystem.java

```diff
@@ -356,15 +420,22 @@ public String getPhysicalDestinationName() {
 
 	/**
 	 * pathname of the file or directory to list.
-	 * @ff.default Home folder of the ftp user
+	 * @ff.default Home folder of the sftp user
 	 */
 	public void setRemoteDirectory(String remoteDirectory) {
 		this.remoteDirectory = remoteDirectory;
+		if (ftpClient != null && ftpClient.isConnected()) {
+			try {
+				ftpClient.cd(remoteDirectory);
+			} catch (SftpException e) {
+				log.warn("unable to change remote directory to [{}]", remoteDirectory, e);
+			}
+		}
```

Comment:
Dit is de hoofd directory, dus alle (relatieve) acties moeten vanaf dit pad gebeuren. Het zetten gebeurd na het inladen van de bean, daarna wordt de configure pas aangeroepen, en de overige acties gebeuren pas runtime.
Hier is dus nog geen ftpClient connectie beschikbaar.

Reply:
Ja, ik begreep dat bij een normale flow dit dan alleen wordt aangeroepen. Maar als je dus `setRemoteDirectory` wil gebruiken om gewoon een `cd dir` te doen, of een use case in de toekomst, dan is dit wel degelijk een goede wijziging. 

In een unit test liep ik hier tegen aan, dat dit gewoon niet werkt. De huidige directory wordt niet bijgehouden verder. Dus bij het inladen heeft dit geen effect, maar wel als het later wordt gebruikt. Misschien heb ik dan een feature gemaakt voor de toekomst (en mijn unittest).

Reply:
Ik snap je argument, maar maak mij wel zorgen omdat dit als argument gebruikt wordt om de client te 'openen' / maken.

Reply:
Zoals zojuist besproken: ik haal dit hier weg en maak er een losse change-directory van. Dat is consistenter.